### ISSUE TITLE
Greyed out items difficult to read in bright light

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#3876]
Hi,
I'm just back from the slope using 1.5.6 for the first time and all is mostly good aside one annoyance. I wanted to make a couple of minor programming changes, however I found that trying to read greyed out lines (LS in this case) virtually impossible in bright light, even with the screen brightness turned fully up. I struggled to see the greyed out lines to know which line I needed to select to edit.

Would it be possible to perhaps make more use of colours to identify which lines are active and which are not please? If necessary, maybe this could be an option so people can play according to their eyesight). I'm not sure which colours would work but I know grey doesn't. I think active lines should be bold as well as a colour and inactive lines not bold and a different  (non-grey) colour?

Many thanks

**Comment by Toffs** [comment]
Would changing the active lines to be the already customisable highlight theme colour work?

**Comment by Jedsters** [comment]
Thanks @Toffs, I did wonder about that, my only concern was whether it would be clear that the existing bold white text was the Active LS (in that example) and the highlight theme colour was the inactive.
You're using a colour as a highlight theme in one instance and then using it to indicate something is inactive elsewhere. 
Would it be better the other way around, e.g. the theme colour bolded to indicate active and unbold white to indicate inactive?

**Comment by Toffs** [comment]
I see what you’re saying. I also spotted this issue post, https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/3522
and wondered if having a quick button to switch between light & dark mode might help with general readability with sun glare perhaps?

**Comment by Jedsters** [comment]
Interesting idea, although it would need to be something which improves the contrast between the medium grey text and slightly darker grey surround. Although I'm already over-loading some of my switches etc.  LOL

**Comment by Jedsters** [comment]
I notice opentx just uses white and bold white which seems to work fine, maybe the easiest solution is just to use a much lighter shade of grey, maybe close to but not quite white.

**Comment by Richrfl** [comment]
I think that Jed's proposal is a simple and good one. One step up would be to add "shade brightness" to the System General tab so that the user can adjust the shades depending on his/her environmental conditions. As per today, reading the shaded text on a sunny day in Florida is impossible, even under a shade. (my yesterday's experience)

**Comment by Jedsters** [comment]
The shade brightness option etc. makes sense - it can then still be referred to as 'greyed out' and we can choose  which shade of grey / how bright.
If it was difficult to read in the UK I can see how Florida in sun would be impossible.  
Thanks

**Comment by Richrfl** [comment]
Gents, I almost accidentally found out a walk around about the shaded settings. Is not the simplest, but it is possible. Let's say that you are in the SF screen, and you have most of the lines shaded. If you start to scroll down, each line will be shown over a background color, which will make them easily readable.

**Comment by Jedsters** [comment]
Indeed, but as you say, very much a work around.  
Thanks for the suggestion though.

**Comment by Richrfl** [comment]
Gents, is there any possibility of producing this enhancement? It's tough to make changes in the field going blind in the list of non-active conditions in the Logical Switch or the Special functions list (and I guess, in other lists as well)

**Comment by Jedsters** [comment]
Still very much wanted, not being able to read things like LS on the slope is one of the things which annoys me most, I'm actually embarrassed to show other people how easy the programming is with Ethos as I know they won't be able to see it!

**Comment by Richrfl** [comment]
I fully agree with Jed. We need this enhancement badly in the land of the sun and hurricanes. Please, developers, put this in the list unless there is a hardware limitation that will prevent it. Don't let us down on this one, please.

**Comment by Jedsters** [comment]
Still very much needed, I struggle to sell my chums the delights of Ethos when they can't even see the screen. It becomes an immediate no thanks.

**Comment by Richrfl** [comment]
> Still very much needed, I struggle to sell my chums the delights of Ethos when they can't even see the screen. It becomes an immediate no thanks.

Jed. After some research, I concluded that this is a hardware-related issue and the Ethos developers cannot do anything about it. From what I read, the only chance to get a slightly brighter screen is to swap the screen for the one used in one of the other X18 (I have not right now if that is the R or some other model, but I think this was treated in the FrSky Official group in Facebook, for someone that had the same problem as we do and found the solution with this other display.

**Comment by Jedsters** [comment]
I'm afraid I disagree. The problem is not the brightness of the screen itself so much as displaying things such as inactive LS  as grey on grey (see below). If one of the greys was a different colour or a colour of our choice then the problem would be resolved. 
Or maybe the active lines are bold white while the inactive lines are non-bold white. Or maybe the active text should be system colour (orange here) and the non-active lines white - difficult to know what works best in bright light without being able to test it. 

![image](https://github.com/user-attachments/assets/5bb46620-0976-4d43-8d7a-95d35297b98e)

**Comment by Richrfl** [comment]
> I'm afraid I disagree. The problem is not the brightness of the screen itself so much as displaying things such as inactive LS as grey on grey (see below). If one of the greys was a different colour or a colour of our choice then the problem would be resolved. Or maybe the active lines are bold white while the inactive lines are non-bold white. Or maybe the active text should be system colour (orange here) and the non-active lines white - difficult to know what works best in bright light without being able to test it.
> 
> ![image](https://private-user-images.githubusercontent.com/96293240/403128102-5bb46620-0976-4d43-8d7a-95d35297b98e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzY4OTgwNjAsIm5iZiI6MTczNjg5Nzc2MCwicGF0aCI6Ii85NjI5MzI0MC80MDMxMjgxMDItNWJiNDY2MjAtMDk3Ni00ZDQzLThkN2EtOTVkMzUyOTdiOThlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAxMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMTE0VDIzMzYwMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjZmQ2MTM2ZmZiYjIyMWRmYmJjZThhOGFjNjJlNDg0MzE3ODU4ZWUxNThmNTNkMTg3MDBiMzU3ZGMyMjhhMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.xGXDJ1qQvkTjinnSE24slQVq8ShMlEQgdAZjvlauhoY)

Sorry, I was going in the wrong direction. In my case, it is very hard to read even the active lines unless you are in a good shade. One of the alternatives I tried and rejected was to reverse to a white background instead of a dark one. In any case, I agree with you that some alternative should be given to the greyed lines, either brilliance or colour background. I second your suggestion for enhancement.

**Comment by Jedsters** [comment]
I don't think we're going to have the same problem as you of not being able to read active lines in England! Well, maybe 1 day a year.  :P
I wonder if a text colour other than white would help you? Again, difficult to know without trying. This does perhaps suggest that some user options for text and background colours could be of use so we can play and find what works best for each of us.

**Comment by Jedsters** [comment]
Maybe a photo or two will demonstrate the problem and these aren't even outdoors. this is just indoors in brightish light, outdoors it is all but impossible to read some items.
OK, a clean of the screen would help a little but they get dirty in the field...

![Image](https://github.com/user-attachments/assets/31df48ca-6816-4a69-ad53-d2a7d9139710)

![Image](https://github.com/user-attachments/assets/3a4d9dbf-6616-473e-b9b7-5c64ba89303d)

Thanks

**Comment by Jedsters** [comment]
I notice that one of the line items for Ethos 26.1 is to improve the contrast. 

I'm not sure this is much better. In the field only items in white will be readable. Rather than using grey text maybe white non-bold text would be better?  

<img width="773" height="458" alt="Image" src="https://github.com/user-attachments/assets/0c270d03-3cf4-4078-b36f-f40cd67d6f4b" />
